odpf/optimus/runtime_service.proto

```diff
@@ -660,4 +665,21 @@ message RegisterJobEventRequest {
     JobEvent event = 4;
 }
 
-message RegisterJobEventResponse {}
\ No newline at end of file
+message RegisterJobEventResponse {}
+
+message GetReplayListRequest {
+    string project_name = 1;
+}
+
+message GetReplayListResponse {
+    repeated ReplaySpec replay_list = 1;
+}
+
+message ReplaySpec {
+    string id = 1;
+    string job_name = 2;
+    string start_date = 3;
```

Comment:
why not `google.protobuf.Timestamp`

Reply:
i'm fine either way, but since start and end date is a date, i figured the response we are sending is already represented as expected, not in protobuf timestamp. any concern on this?

Reply:
discussed this and decided to use timestamp